main.js

```diff
@@ -0,0 +1,70 @@
+// Declare variables
+let cards = ['Prague','Prague', 'London', 'London', 'Paris', 'Paris', 'Moscow', 'Moscow', 'Los Angeles', 'Los Angeles', 'Vancouver', 'Vancouver', 'Sydney', 'Sydney', 'Helsinki', 'Helsinki', 'Singapore', 'Singapore', 'Tokyo', 'Tokyo'];
+let currentCards = [];
+let points = 0;
+let firstCard, secondCard;
+let flippedCards = 0;
+
+// Shuffle cards
+function shuffle(array) {
+  for (let i = array.length - 1; i > 0; i--) {
+    const j = Math.floor(Math.random() * (i + 1));
+    [array[i], array[j]] = [array[j], array[i]];
+  }
+  return array;
+}
+
+// Create the board
+function createGameBoard() {
+  let gameField = document.getElementById('game-field');
+  let shuffledCards = shuffle(cards);
+  for (let i = 0; i < cards.length; i++) {
+    let card = document.createElement('div');
+    card.classList.add('card');
+    card.dataset.name = shuffledCards[i];
+    card.addEventListener('click', flipCard);
+    gameField.appendChild(card);
+    currentCards.push(card);
+  }
+}
+
+// Flip cards
+function flipCard() {
+  if (flippedCards === 0) {
+    firstCard = this;
+    firstCard.innerHTML = firstCard.dataset.name;
+    firstCard.classList.add('flipped');
+    flippedCards++;
+  } else {
+    secondCard = this;

```

Comment:
muzete zakazat otaceni treti karty pomoci podminky...

Reply:
Založil jsem nový pull request, jestli to není správně, tak nevím už kam.
Když jsem zkoušel zakázat otáčení třetí karty podle Vašeho komentáře, 
tak přišli horší problémy, jako to že pak nešli otáčet karty vůbec, otáčeli se zpátky, 
když byla otočena jen 1, zůstávali otočené nebo jich šlo otočit ještě víc.
Dosáhnu tímto alespoň na 0,5 bodu?